src/einsteinpy/symbolic/ricci.py

```diff
@@ -22,6 +23,10 @@ def __init__(self, arr, syms, config="ll", parent_metric=None):
             Tuple of crucial symbols dentoting time-axis, 1st, 2nd, and 3rd axis (t,x1,x2,x3)
         config : str
             Configuration of contravariant and covariant indices in tensor. 'u' for upper and 'l' for lower indices. Defaults to 'll'.
+        parent_metric : ~einsteinpy.symbolic.metric.MetricTensor or None
+            Corrosponding Metric for the Ricci Tensor.
+            None if it should inherit the Parent Metric of Christoffel Symbols.
```

Comment:
Remove this line `None if it should inherit the Parent Metric of Christoffel Symbols.`

Reply:
Also one more thing, add a `@property`, `parent_metric` to `RicciTensor` class.
Take reference from `RiemannCurvatureTensor` class!